lib/lorekeeper/json_logger.rb

```diff
@@ -113,18 +118,38 @@ def json_message(message)
 
     # Some instrumentation libraries pollute the stacktrace and create a large output which may
     # cause problems with certain logging backends.
-    # Hardcording newrelic, active_support/callbacks and zipkin-tracer now here.
+    # Hardcording newrelic, active_support/callbacks, zipkin-tracer, web servers and stdlib now here.
     # In the future if this list grows, we may make it configurable.
     def clean_backtrace(backtrace)
+      backtrace = filter_rails_root_backtrace(backtrace)
       @backtrace_cleaner&.clean(backtrace) || backtrace
     end
 
+    def filter_rails_root_backtrace(backtrace)
```

Comment:
Is there a spec that illustrates how this function works?  Specifically to show what lines are removed that would not be removed by the cleaner.

Reply:
like this 5bbf92e3a0f82614809d5fcd0b867fba6fb317a0?